modin/logging/class_logger.py

```diff
@@ -53,5 +55,7 @@ def __init_subclass__(
             The log level (INFO, DEBUG, WARNING, etc.).
         **kwargs : dict
         """
+        modin_layer = modin_layer or cls._modin_logging_layer
         super().__init_subclass__(**kwargs)
         enable_logging(modin_layer, class_name, log_level)(cls)
+        cls._modin_logging_layer = modin_layer
```

Comment:
Does this change the `_modin_logging_layer` for all children of `ClassLogger` or just the children of `ClassLogger` and class `Foo(ClassLogger)`? For example, if there are classes:

`Foo(ClassLogger, modin_layer="Custom")`, `Bar(Foo)` and `Other(ClassLogger)`, is `Other` guaranteed to have `modin_layer="PANDAS-API"`?